baybe/constraints/discrete.py

```diff
@@ -183,7 +185,10 @@ def get_invalid(self, data: pd.DataFrame) -> pd.Index:  # noqa: D102
             [
                 censored_data[other_params],
                 censored_data[all_affected_params].apply(
-                    frozenset if self.permutation_invariant else tuple, axis=1
+                    cast(Callable, frozenset)
```

Comment:
yeah it must be about something with this builtin rather than a custom function and/or the fact that pandas typing seems still incomplete :/

apart from the the fix here should be fine, I know its ugly but fine for me

Reply:
Ok, then let’s blame pandas and keep it 👍🏼